src/wallet/gtest/test_wallet.cpp

```diff
@@ -99,7 +99,7 @@ CWalletTx GetValidReceive(const libzcash::SpendingKey& sk, CAmount value, bool r
     // Prepare JoinSplits
     uint256 rt;
     JSDescription jsdesc {*params, mtx.joinSplitPubKey, rt,
-                          inputs, outputs, value, 0, false};
+                          inputs, outputs, 2*value, 0, false};
```

Comment:
Is this (and subsequent changes in this file) for fixing issues that are now properly caught? If so, nice!